server/api/v1/db/service/dbservice.js

```diff
@@ -0,0 +1,8 @@
+const mongoose = require('mongoose');
```

Comment:
This file should be under server/services/mongo/mongo.service.js

Reply:
Also, create a Class called MongoService. don't just export the function use inside a class.